opm/autodiff/ThreadHandle.hpp

```diff
@@ -150,8 +150,6 @@ namespace Opm
         if( createThread )
```

Comment:
The two member variables should still be switched, since thread uses a reference of ThreadHandleQueue and thus should be deleted after ThreadHandleQueue